problem-gson/src/test/java/org/zalando/problem/OutOfStockException.java

```diff
@@ -0,0 +1,42 @@
+package org.zalando.problem;
+
+import java.net.URI;
+
+import static org.zalando.problem.Status.BAD_REQUEST;
+
+class OutOfStockException extends ThrowableProblem {
+
+    private static final String TYPE_NAME = "https://example.org/out-of-stock";
+    static final URI TYPE = URI.create(TYPE_NAME);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/whiskeysierra/problem/pullRequest?prid=4476243)